erpnext/templates/pages/order.py

```diff
@@ -11,6 +11,7 @@ def get_context(context):
 	context.no_cache = 1
 	context.show_sidebar = True
 	context.doc = frappe.get_doc(frappe.form_dict.doctype, frappe.form_dict.name)
+	context.payment_gateway_list = frappe.get_doc("Shopping Cart Settings", "Shopping Cart Settings").as_dict()["gateways"]
```

Comment:
Breaking it into multiple variables will help in debugging

```suggestion
	cart_settings = frappe.get_single_doc("Shopping Cart Settings")
	context.payment_gateway_list = cart_settings.gateways
```